packages/webview_flutter/example/lib/main.dart

```diff
@@ -47,3 +62,40 @@ class SampleMenu extends StatelessWidget {
     );
   }
 }
+
+class MoveControl extends StatelessWidget {
+  const MoveControl(this._webViewController)
+      : assert(_webViewController != null);
+
+  final WebViewController _webViewController;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        IconButton(
+            icon: const Icon(Icons.arrow_back_ios),
+            onPressed: () async {
+              final bool canGoBack = await _webViewController.canGoBack;
+              if (canGoBack) {
+                await _webViewController.goBack();
+              } else {
+                Scaffold.of(context).showSnackBar(
+                    const SnackBar(content: Text("Can't go back")));
+              }
+            }),
+        IconButton(
+            icon: const Icon(Icons.arrow_forward_ios),
+            onPressed: () async {
+              final bool canGoForward = await _webViewController.canGoForward;
+              if (canGoForward) {
+                await _webViewController.goForward();
+              } else {
+                Scaffold.of(context).showSnackBar(
+                    const SnackBar(content: Text("Can't go forward")));
```

Comment:
nit: I'd change the message to "No forward history item".

Reply:
Fixed on 59646c8